### PR TITLE
adding contribution file

### PR DESCRIPTION
**Pull request by AEM100**
I have created the contribution file for you, see if you like it or want me to change or add something

image

### CODE REVIEW

**Comment by ThePrez** [comment]
@AEM100 thanks for the organized contributions file! Can you please change the build steps from gradlew to mvn?

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+
+## Making Changes
+Make your changes to the codebase. Ensure that your changes adhere to our [style guide](#style-guide).
+
+## Committing Changes
+Commit your changes with a descriptive commit message. Follow the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) specification for your commit messages.
+
+
+git add .
+git commit -m "feat: add new feature description"
```
Comment (SanjulaGanepola) [inline CONTRIBUTING.md:72]: This should be wrapped in a `sh` code block and should include the `-s` option to sign commits:

```sh
git add .
git commit -s -m "feat: add new feature description"
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+
+### Reporting Bugs
+
+If you find a bug, please report it by opening an issue on our [GitHub Issues page](https://github.com/your-username/mapepire-java/issues). Provide as much detail as possible, including steps to reproduce the issue, your environment, and any relevant logs.
+
+Before you start working on a bug, please comment on the issue to let us know that you are working on it. This helps prevent duplicate efforts.
+
+### Suggesting Enhancements
+
+We welcome new ideas! If you have a feature request or enhancement suggestion, please open an issue on our [GitHub Issues page](https://github.com/your-username/mapepire-java/issues) and describe the enhancement you'd like to see.
```
Comment (SanjulaGanepola) [inline CONTRIBUTING.md:36]:
```suggestion
We welcome new ideas! If you have a feature request or enhancement suggestion, please open an issue on our [GitHub Issues page](https://github.com/Mapepire-IBMi/mapepire-java/issues) and describe the enhancement you'd like to see. 
```

**Review by SanjulaGanepola — CHANGES_REQUESTED** [review]
Thanks for adding this file! Just a few minor comments

**Review by SanjulaGanepola — APPROVED** [review]
Merging this PR and I will make the relevant fixes in another branch.

**Pull request merged**